scripts/promote-tracks.py

```diff
@@ -0,0 +1,84 @@
+import requests
+import datetime
+from dateutil import parser
+import os
+
+SNAPSTORE_API = "https://api.snapcraft.io/v2/snaps/info/"
+PROMOTE_API_URL = "https://api.snapcraft.io/v2/snaps/revisions/promote"
+SNAP_NAME = "k8s"
+IGNORE_TRACKS = ["latest"]
+
+# Headers for Snap Store API request
+HEADERS = {
+    'Snap-Device-Series': '16',
+    'User-Agent': 'Mozilla/5.0',
+}
+
+# Get the authorization token
+AUTH_TOKEN = os.getenv("SNAPSTORE_AUTH_TOKEN")
+
+# Headers for authenticated API requests
+AUTH_HEADERS = {
+    'Authorization': f'Bearer {AUTH_TOKEN}',
+    'Content-Type': 'application/json',
+}
+
+RISK_LEVELS = ["edge", "beta", "candidate", "stable"]
+
+DAYS_TO_STAY_IN_RISK = {
+    "edge": 1,
+    "beta": 3,
+    "candidate": 5
+}
+
+def get_snap_info(snap_name):
+    response = requests.get(SNAPSTORE_API + snap_name, headers=HEADERS)
+    response.raise_for_status()
+    return response.json()
+
+def promote_revision(revision, channel, dry_run=True):
+    if not dry_run:
+        payload = {
+            "actions": [
+                {
+                    "action": "release",
+                    "revision": revision,
+                    "channels": [channel]
+                }
+            ]
+        }
+        response = requests.post(PROMOTE_API_URL, headers=AUTH_HEADERS, json=payload)
+        response.raise_for_status()
+    print(f"Successfully promoted revision {revision} to {channel}")
+
+
+def check_and_promote(snap_info):
+    channels = {c["channel"]["name"]: c for c in snap_info["channel-map"]}
+
+    for channel_info in snap_info['channel-map']:
+        channel = channel_info["channel"]
+        track = channel["track"]
+        risk = channel["risk"]
+        next_risk = RISK_LEVELS[RISK_LEVELS.index(risk)+1]
+        revision = channel_info["revision"]
+
+        if track in IGNORE_TRACKS:
+            continue
+
+        now = datetime.datetime.now(datetime.timezone.utc)
+
+        if ((now - parser.parse(channel["released-at"])).days > DAYS_TO_STAY_IN_RISK[risk]
```

Comment:
`dateutil` is not standard library either. instead of relying on that, can we use `datetime.strptime` instead?